utils/__mocks__/webpackCompiler.js

```diff
@@ -1,13 +1,37 @@
 const run = jest.fn();
-const options = {
-  entry: [],
+
+const defaultOptions = {
+  entry: {
+    main: ['main.js'],
+  },
   devServer: 'devServer',
   output: {
     path: 'fakePath',
     publicPath: 'publicPath',
   },
 };
+
+const multiEntryOptions = {
+  entry: {
+    main: ['main.js'],
+    additional: ['more.js'],
+  },
```

Comment:
Ah, I see. Maybe adding the test case with a string literal would be useful though.